wallet/src/ui/app/ApiProvider.ts

```diff
@@ -54,6 +55,14 @@ export default class ApiProvider {
         this._apiProvider = new JsonRpcProvider(DEFAULT_API_ENDPOINT);
     }
 
+    public setNewJsonRpcProvider(apiEnv: string) {
```

Comment:
Changing `apiEnv` from `string` to `API_ENV` should make more clear that only `API_ENV` is supported

Reply:
👍